### ISSUE TITLE
ASRock PPT setting, screenshot

### ISSUE THREAD

**Issue by radianttap** [nickbabcock/sff.life#372]
![CCF7AB44-0E8C-48F0-BBE8-E065A118FBCD](https://user-images.githubusercontent.com/80697/78463965-d87dd380-76e3-11ea-9582-665b53809cc3.jpeg)

**Comment by radianttap** [comment]
You asked for screenshot, so here it is. 
I first tested using Ryzen Master and found that with 60W PPT I’m losing fairly little performance. 88W is default for PPT. 

Cinebench R20 score for Ryzen 5 3600 went from 3532 to 3318. Max temp from 84 to 67°C. Sustained core clock from 3875 to 3675 MHz. Blender BMW render from 4:01 to 4:13, mere 12s more. 

Sounds like a win to me.

**Comment by nickbabcock** [comment]
Excellent, always happy to hear another win for undervolting. I've updated the site with your screenshot. Let me know if you want to be credited a different way. Thank you!

**Comment by radianttap** [comment]
When you get to it, only put my name (Aleksandar Vacić) instead of the handle.
Thank you for the great guide, very well and very clearly written. I learned a lot.

Here's the table with the results I got, while testing. Not totally thorough as you recommended (did not run 3 cycles for each PPT change) but it was enough for me to see the trend line.

![IMG_0078](https://user-images.githubusercontent.com/80697/78469929-b311b980-7725-11ea-853d-1c8c55c05aac.jpeg)
